lecli/log_api.py

```diff
@@ -0,0 +1,174 @@
+"""
+Log API module.
+"""
+import json
+import requests
+
+from lecli import api_utils
+from lecli import response_utils
+
+def _url():
+    """
+    Get rest query url of log resource id.
+    """
+    return 'https://rest.logentries.com/management/logs'
+
+
+def handle_get_log_response(response):
+    """
+    Handle get log response.
+    """
+    if response_utils.response_error(response):
+        exit(1)
+    elif response.status_code == 200:
+        if 'log' in response.json():
```

Comment:
@stopal-r7 your is a little bit dense and i haven't seen that pattern before that much in the python world.